napari/_qt/layer_controls/qt_vectors_controls.py

```diff
@@ -126,6 +119,16 @@ def __init__(self, layer):
         self.layout().addRow(self.edge_prop_label, self.color_prop_box)
         self.layout().addRow(trans._('out of slice:'), self.outOfSliceCheckBox)
 
+        self.layer.events.edge_width.connect(self._on_edge_width_change)
```

Comment:
Was the position of this actually causing issues? If so, you should add an explanatory implementation comment.

Or do you just prefer these layer event connections to go last? If so, no need to do anything.